Add Hindi Discourse Analysis Natural Language Inference Dataset (#1822)
huggingface/datasets: datasets/hda_nli_hindi/README.md

---
annotations_creators:
- machine-generated
language_creators:
- found
languages:
- hi
licenses:
- mit
multilinguality:
- monolingual
size_categories:
- 10K<n<100K
source_datasets:

> - extended|hindi_discourse_analysis_classification

Reviewer: ```suggestion
- extended|hindi_discourse
```
It's the correct name in the `datasets` library.

task_categories:
- text-classification
task_ids:
- natural-language-inference
---

# Dataset Card for Hindi Discourse Analysis Dataset

## Table of Contents
- [Dataset Description](#dataset-description)
  - [Dataset Summary](#dataset-summary)
  - [Supported Tasks](#supported-tasks-and-leaderboards)
  - [Languages](#languages)
- [Dataset Structure](#dataset-structure)
  - [Data Instances](#data-instances)
  - [Data Fields](#data-fields)
  - [Data Splits](#data-splits)
- [Dataset Creation](#dataset-creation)
  - [Curation Rationale](#curation-rationale)
  - [Source Data](#source-data)
  - [Annotations](#annotations)
  - [Personal and Sensitive Information](#personal-and-sensitive-information)
- [Considerations for Using the Data](#considerations-for-using-the-data)
  - [Social Impact of Dataset](#social-impact-of-dataset)
  - [Discussion of Biases](#discussion-of-biases)
  - [Other Known Limitations](#other-known-limitations)
- [Additional Information](#additional-information)
  - [Dataset Curators](#dataset-curators)
  - [Licensing Information](#licensing-information)
  - [Citation Information](#citation-information)
  - [Contributions](#contributions)

## Dataset Description

> - HomePage : https://github.com/midas-research/hindi-nli-data
> - Paper : https://www.aclweb.org/anthology/2020.aacl-main.71
> - Point of Contact : https://github.com/midas-research/hindi-nli-data

Reviewer: ```suggestion
- **HomePage:** [GitHub](https://github.com/midas-research/hindi-nli-data)
- **Paper:** [Aclweb](https://www.aclweb.org/anthology/2020.aacl-main.71)
- **Point of Contact:** [GitHub](https://github.com/midas-research/hindi-nli-data)
```
this is the formatting we usually use

### Dataset Summary

- Dataset for Natural Language Inference in Hindi Language. Hindi Discourse Analysis (HDA) Dataset consists of textual-entailment pairs.
- Each row of the Datasets if made up of 4 columns - Premise, Hypothesis, Label and Topic.
- Premise and Hypothesis is written in Hindi while Entailment_Label is in English.
- Entailment_label is of 2 types - entailed and not-entailed.
- Entailed means that hypotheis can be inferred from premise and not-entailed means vice versa
- Dataset can be used to train models for Natural Language Inference tasks in Hindi Language.

### Supported Tasks and Leaderboards

- Natural Language Inference for Hindi

### Languages

- Dataset is in Hindi

## Dataset Structure

- Data is structured in TSV format. 
- train, test and dev files are in seperate files

### Dataset Instances

An example of 'train' looks as follows.

```
{'hypothesis': 'यह एक वर्णनात्मक कथन है।', 'label': 1, 'premise': 'जैसे उस का सारा चेहरा अपना हो और आँखें किसी दूसरे की जो चेहरे पर पपोटों के पीछे महसूर कर दी गईं।', 'topic': 1}

```
### Data Fields

> - Each row contatins 4 columns - premise, hypothesis, label and topic.

Reviewer: ```suggestion
Each row contatins 4 columns:
- premise: string
- hypothesis: string
- label: class label with values that correspond to "not-entailment" (0) or "entailment" (1)
- topic: class label with values that correspond to "Argumentative" (0), "Descriptive" (1), "Dialogic" (2), "Informative" (3) or "Narrative" (4).
```
we can show the label names here
